src/test/java/PlayListTests.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlayListTests extends BaseTest{
+    @Test
+    public void addSongToPlaylist()throws InterruptedException {
+        login("chernyakma75@gmail.com", "te$t$tudent");
+
+        WebElement SearchField= driver.findElement(By.cssSelector("[type='search']"));
+        SearchField.click();
+        SearchField.clear();
+        SearchField.sendKeys("Dark days");
+
+        WebElement viewAllButton = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
+        viewAllButton.click();
+
+        WebElement FirstSongButton = driver.findElement(By.cssSelector("#songResultsWrapper .items > tr:nth-of-type(1) > .title"));
+        FirstSongButton.click();
+
+        WebElement addToButton = driver.findElement(By.cssSelector(".btn-add-to"));
+        addToButton.click();
+
+        WebElement ChoosePlaylistButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']//li[contains(text(),'P1')] "));
+        ChoosePlaylistButton.click();
+
+        WebElement Notification = driver.findElement(By.cssSelector(".success.show"));
+        Assert.assertTrue(Notification.isDisplayed());}
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._